src/components/BlogCard/BlogCard.scss

```diff
@@ -0,0 +1,56 @@
+@import "./../../styles/variables";
+
+.blog-card {
+  border-radius: 0.5rem;
+  border: 1px solid $black-color-200;
+  margin-bottom: 1rem;
+  padding: 1rem;
+  &__banner {
+    border-radius: 0.5rem;
+    height: 15rem;
+    width: 100%;
+  }
+
+  &__category-badge {
+    background-color: $secondary-color;
+    border-radius: 0.5rem;
+    color: $color-blue;
+    margin: 1rem;
+    padding: 0.5rem;
+    width: 9.375rem;
+  }
+
+  &__blog-title {
+    -webkit-box-orient: vertical;
+    -webkit-line-clamp: 3;
+    display: -webkit-box;
```

Comment:
could you please explain?

Reply:
These CSS properties are used for controlling the layout and presentation of text within a container.